### PR TITLE
Translation of the concurrency.md file

### PR DESCRIPTION
**Pull request by getbrandao**
Translation of the concurrency.md file in pt/lessons/advanced to Portuguese.

### CODE REVIEW

File: pt/lessons/advanced/concurrency.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+---
+layout: página
+title: Concorrência
+category: avançado
+order: 4
+lang: pt
+---
+
+Um dos pontos de oferta do Elixir é o suporte a concorrência. Obrigado à ErlangVM, concorrência no Elixir é mais fácil do que o esperado. O modelo de concorrência replica sobre Atores, um processo constante que se comunica com outros processos através de passagem de mensagem.
```
Comment (pragmaticivan) [inline pt/lessons/advanced/concurrency.md]: `Um dos pontos ofertados`

Comment (pragmaticivan) [inline pt/lessons/advanced/concurrency.md]: `Graças à ErlangVM`

Comment (pragmaticivan) [inline pt/lessons/advanced/concurrency.md]: `do que esperamos`

Comment (getbrandao) [inline pt/lessons/advanced/concurrency.md]: In the case I have to do a new request pull, after changes?

File: pt/lessons/advanced/concurrency.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+- [Processos](#processos)
+  - [Passagem de mensagens](#passagem-mensagem)
+  - [Vinculando Processos](#vinculando-processos)
+  - [Monitorando Processos](#monitorando-processos)
+- [Agentes](#agentes)
+- [Tarefas](#tarefas)
+
+## Processos
+
+Processos no ErlanVM são leves e executam em todas CPUs. Enquanto eles podem parecer como threads nativas, eles são bastantes simples e não é incomum ter milhares de processos concorrentes em uma aplicação Elixir.
```
Comment (pragmaticivan) [inline pt/lessons/advanced/concurrency.md]: `em todas as CPUs`

File: pt/lessons/advanced/concurrency.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+A forma mais fácil para cirar um novo processo é o `spawn` na qual tem tanto uma função nomeada ou anônima. Quando criamos um novo processo ele retorna um _Process Identifier_ ou PID, para exclusivamente identificá-lo dentro de nossa aplicação.
```
Comment (pragmaticivan) [inline pt/lessons/advanced/concurrency.md]: para `criar`

File: pt/lessons/advanced/concurrency.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+end
+
+iex> Example.run
+Exit reason: kaboom
+:ok
+```
+
+## Agentes
+
+Agentes são uma abstração acerca processos em segundo plano em estado de manutenção. Podemos acessa-los de outros processo dentro de nossa aplicação ou nó. O estado do nosso Agente definido como valor de retorno de nossa função:
```
Comment (pragmaticivan) [inline pt/lessons/advanced/concurrency.md]: `acerca de processos`

Comment (pragmaticivan) [inline pt/lessons/advanced/concurrency.md]: `de outros processos dentro`

Comment (pragmaticivan) [inline pt/lessons/advanced/concurrency.md]: `do nosso Agente é definido como valor`

File: pt/lessons/advanced/concurrency.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+iex> Agent.start_link(fn -> [1, 2, 3] end, name: Numbers)
+{:ok, #PID<0.74.0>}
+
+iex> Agent.get(Numbers, &(&1))
+[1, 2, 3]
+```
+
+## Tarefas
+
+Tarefas fornece uma forma para executar uma função em segundo plano e posteriormente recuperar seu valor. Elas podem ser particularmente útil ao manusear operações dispendiosa, sem bloquear a execução do aplicativo.
```
Comment (pragmaticivan) [inline pt/lessons/advanced/concurrency.md]: `Tarefas fornecem`

**Comment by pragmaticivan** [comment]
@getbrandao just left some comments, thanks!

File: pt/lessons/advanced/concurrency.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+category: avançado
+order: 4
+lang: pt
+---
+
+Um dos pontos ofertados pelo Elixir é o suporte a concorrência. Graças à ErlangVM, concorrência no Elixir é mais fácil do que esperamos. O modelo de concorrência replica sobre Atores, um processo constante que se comunica com outros processos através de passagem de mensagem. 
+
+Nesta aula nós veremos os módulos de concorrência que vêm com Elixir. No próximo capítulo nós cobriremos os comportamentos OTP que os implementam.
+
+## Tabela de Conteúdos
```
Comment (pragmaticivan) [inline pt/lessons/advanced/concurrency.md]: Replace this for `Sumário`

**Comment by pragmaticivan** [comment]
@getbrandao please rebase from master and squash your commits. Let me know if you need any help.

**Comment by getbrandao** [comment]
@pragmaticivan  squash from master is done!

**Comment by doomspork** [comment]
Howdy @getbrandao, do you use Slack?  I can walk you through the rebase if you're free.

**Comment by pragmaticivan** [comment]
@getbrandao we have a little bit of messy here. What I recommend here is. Copy the content of the translation.  Pull master from upstream. Create another branch from master and open a new pull request with the content. You can close this Pull Request here.

**Comment by getbrandao** [comment]
@doomspork, yes I use Slack. Would you mind help me?

**Pull request closed without merging**